src/Components/Programs/Timeline.js

```diff
@@ -58,14 +58,14 @@ function Timeline() {
                                     color:item.color,
                                     borderColor:item.color,
                                     flex:1,
-                                    left:item.date[0][2]*10,
+                                    left:50,
```

Comment:
@nlok5923  Remove the 'left' property from here and see if it fixes the overlapping issue. Otherwise adjust the width of the 'ScrollContainer'.

Reply:
removing left property will make event components stick leftwards which won't look good as well as issue won't be resolved with that approach